src/status_im/contexts/profile/settings/events.cljs

```diff
@@ -139,10 +139,33 @@
             :on-success #(log/debug "mnemonic was marked as shown")
             :on-error   #(log/error "mnemonic was not marked as shown" %)}]]]}))
 
+(defn token-symbols-from-accounts
+  [accounts]
+  (->> accounts
+       vals
+       (reduce (fn [acc account]
+                 (->> account
+                      :tokens
+                      (map :symbol)
+                      (into acc)))
+               #{})
+       vec))
```

Comment:
Is there a reason the unordered sequence of unique symbols must be a vector and not a set?

Reply:
Thank you @vkjr, sorry for my delays just now seeing your updates in this PR.